src/lib/mongoBackend/MongoCommonSubscription.cpp

```diff
@@ -359,3 +360,15 @@ void setFormat(const Subscription& sub, BSONObjBuilder* b)
   b->append(CSUB_FORMAT, format);
   LM_T(LmtMongo, ("Subscription format: %s", format.c_str()));
 }
+
+/* ****************************************************************************
+*
+* setBlacklist -
+*
+*/
+void setBlacklist(const Subscription& sub, BSONObjBuilder* b)
+{
+  bool bl = sub.notification.blacklist;
+  b->append(CSUB_BLACKLIST, bl);
+  LM_T(LmtMongo, ("Subscription blacklist: %s", bl?"TRUE":"FALSE"));
```

Comment:
I think in other places (eg. extended) we are using "true" and "false" (lowercase) so maybe it is better that way, to align.

In addition, I think that whitespace would be added for `?` and `:` (not sure 100%).